_data-prepper/core-apis.md

```diff
@@ -0,0 +1,104 @@
+git ---
+layout: default
+title: Core APIs
+nav_order: 2
+---
+
+All Data Prepper instances expose a server with some control APIs. By default, this server runs on port 4900. Some plugins, especially Source plugins may expose other servers. These will be on different ports and their configurations are independent of the core API. For example, to shut down Data Prepper, you can run:
+
+```
+curl -X POST http://localhost:4900/shutdown
+```
+
+# APIs
+
+The following APIs are available:
```

Comment:
@dlvenable I think this section would benefit from being formatted as a table to make it easier to see the information at a glance, and because it follows a title/definition format.